gensim/models/nmf.py

```diff
@@ -45,25 +79,26 @@ def __init__(
         sparse_coef=3,
         random_state=None,
     ):
-        """
+        r"""
 
         Parameters
         ----------
         corpus : iterable of list of (int, float), optional
-            Training corpus. If not given, model is left untrained.
+            Training corpus. Contains list of counts of words for every document.
         num_topics : int, optional
             Number of topics to extract.
-        id2word: gensim.corpora.Dictionary, optional
+        id2word: gensim.corpora.dictionary.Dictionary, optional
             Mapping from token id to token. If not set words get replaced with word ids.
         chunksize: int, optional
             Number of documents to be used in each training chunk.
-        passes: int, optioanl
+        passes: int, optional
             Number of full passes over the training corpus.
-        lambda_ : float, optional
+        \lambda_ : float, optional
             Residuals regularizer coefficient. Increasing it helps prevent ovefitting. Has no effect if `use_r` is set
             to False.
         kappa : float, optional
-            Optimizer step coefficient. Increaing it makes model train faster, but adds a risk that it won't converge.
+            Gradient descent step size.
```

Comment:
missing useful ranges for parameters (here and everywhere)

Reply:
You're right, added that to TODO